src/input/hotkey.c

```diff
@@ -160,10 +166,58 @@ static void input_number(int number)
     }
 }
 
+static void new_map(void)
+{
+    if (editor_is_active() && window_is(WINDOW_EDITOR_MAP)) {
+        // TODO: here we need the size selection for triggering window_select_list_show?
+    } else if (window_is(WINDOW_CITY)) {
+        building_construction_clear_type();
+        game_undo_disable();
+        game_state_reset_overlay();
+        window_main_menu_show(1);
+    }
+}
+
+static void load_file(void)
+{
+    if (editor_is_active() && window_is(WINDOW_EDITOR_MAP)) {
```

Comment:
No need to check for `editor_is_active()`, since you can  only be on the `WINDOW_EDITOR_MAP` window when the editor is active.